sdk/monitorexporter/azure-analytics-monitorexporter/src/main/java/com/azure/analytics/monitorexporter/MonitorExporterAsyncClient.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.analytics.monitorexporter;
+
+import com.azure.analytics.monitorexporter.implementation.ApplicationInsightsClientImpl;
+import com.azure.analytics.monitorexporter.models.TelemetryItem;
+import com.azure.analytics.monitorexporter.models.TrackResponse;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+/**
+ * This class contains synchronous operations to interact with the Azure Monitor Exporter service.
+ */
+@ServiceClient(builder = MonitorExporterClientBuilder.class, isAsync = true)
+public class MonitorExporterAsyncClient {
+    private final ApplicationInsightsClientImpl restServiceClient;
+
+    MonitorExporterAsyncClient(ApplicationInsightsClientImpl restServiceClient) {
+        this.restServiceClient = restServiceClient;
+    }
+
+    /**
+     * The list of telemetry items that will be sent to the Azure Monitor Exporter service. The response contains the
+     * status of number of telemetry items successfully accepted and the number of items that failed along with the
+     * error code for all the failed items.
+     *
+     * @param telemetryItems The list of telemetry items to send.
+     * @return The response containing the number of successfully accepted items and error details of items that were
+     * rejected.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<TrackResponse> track(List<TelemetryItem> telemetryItems) {
+        return restServiceClient.trackAsync(telemetryItems);
+    }
+
+    /**
+     * The list of telemetry items that will be sent to the Azure Monitor Exporter service. The response contains the
+     * status of number of telemetry items successfully accepted and the number of items that failed along with the
+     * error code for all the failed items.
+     *
+     * @param telemetryItems The list of telemetry items to send.
+     * @return The response containing the number of successfully accepted items and error details of items that were
+     * rejected.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<TrackResponse>> trackWithResponse(List<TelemetryItem> telemetryItems) {
```

Comment:
Should we consider renaming `TrackResponse`  to not have the suffix `Response`. The return type currently might be confusing to the users to have `Response<TrackResponse>`.

How about `TrackMeta`, `TrackStatistics` or `TrackProperties`?

Reply:
`TrackResult`?